### ISSUE TITLE
Serial chart breaks on hover when using latest version (3.18.3)

### ISSUE THREAD

**Issue by vedraan** [amcharts/amcharts3#10]
Using the latest version (3.18.3) I get this error in the console as soon as I hover a chart stick (I'm using candlestick graph type):
```
Uncaught TypeError: a.indexOf is not a function
e.AmSerialChart.e.Class.formatString @ serial.js:46
d.AmGraph.d.Class.showGraphBalloon @ amcharts.js:199
e.AmSerialChart.e.Class.handleCursorMove @ serial.js:58
d.Class.a.inherits.b.fire @ amcharts.js:3
d.ChartCursor.d.Class.dispatchMovedEvent @ amcharts.js:220
d.ChartCursor.d.Class.update @ amcharts.js:206
e.AmRectangularChart.e.Class.update @ serial.js:19
e.AmSerialChart.e.Class.update @ serial.js:52
d.update @ amcharts.js:12
```
I know this isn't much help to you since it's from a minified file, but you will probably figure it out when you test it yourselves. When I downloaded an earlier version (3.17.3) which is what I'm using now to solve this, there are no problems, so it's something that got broken after that version.

If I get more time, I might test the new version some more and post details, but I'm super-busy at the moment so I doubt it will happen any time soon.

**Comment by martynasma** [comment]
I can't seem to be able to reproduce this with the candlestick demos that we have. I.e. this one:

https://www.amcharts.com/demos/candlestick-chart/

Maybe you could share your particular chart setup so we can check?

You can contact us directly at contact@amcharts.com if you want to keep it private.

Thanks!

**Comment by vedraan** [comment]
Hi again, been a while but I finally got the time to create a setup to reproduce this:

https://jsfiddle.net/u86rLqkb/

Since I did not want the balloons on my second graph I used:
`'balloonText': false`
and the above mentioned Uncaught TypeError happened in the new versions. 

I see now in the docs that this is invalid value for this option (string is expected), but for some reason this worked with version 3.17.3.

Since this is probably not a bug then, feel free to close this issue, but a quick question first. 

Is this the best option to stop balloons from showing up, set on just one of the graphs:
```
"balloon": {
      "enabled": false
}
```

Thanks!
